app/controllers/samples_controller.rb

```diff
@@ -228,7 +228,6 @@ def heatmap
         "NT.r",
```

Comment:
We should remove the NT too for the aggregate score.
The plan here is to send a full object: `[{name: ..., value: ...}]` so that metrics show up properly and values are still understanded by the server.

Reply:
fixed